libs/wordpress-plugin/plugin/trunk/block/block.json

```diff
@@ -3,6 +3,7 @@
   "name": "blockprotocol/block",
   "category": "blockprotocol",
   "icon": "schedule",
+  "apiVersion": 2,
```

Comment:
I think this is inferred anyway but the Gutenberg team is suggesting that the iFrame will only be activated in WP Core for pages where all blocks are apiVersion 3+, so I'm making it explicit here.